### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#182] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-13 194645](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/30d76f91-534f-47da-ba8d-fa34249fa57c)

**Comment by CiottoloMaggico** [comment, marked as answer]
## Soluzione confermata in classe
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/41151e84-1a7a-4924-93ff-f7a393b96e1b)

**Comment by daromacarlo** [comment]
ho risolto l'esercizio in maniera diversa, qualcuno sa dirmi se è giusto?
![Immagine 2023-12-11 131449](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/5d35e118-1798-496d-b622-6d35098c6fae)
![Immagine 2023-12-11 131459](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/dfe1e61e-db72-4e9d-b66c-7b194528a009)
